### ISSUE TITLE
请问zhihu_textcnn训练时的loss是交叉熵吗？

### ISSUE THREAD

**Issue by coderSkyChen** [Qinbf/Tensorflow#3]
tf.nn.softmax_cross_entropy_with_logits   训练时loss不降反升，不能收敛

**Comment by Qinbf** [comment]
用tf.nn.sigmoid_cross_entropy_with_logits

**Comment by bluekingdom** [comment]
@coderSkyChen softmax loss 是用于单目标分类，即 num_class 中 选 1 个。但标签中有多个 1。 所以loss不收敛。
